lib/MarcVersionHistory/MarcVersionHistory.js

```diff
@@ -51,8 +51,9 @@ const MarcVersionHistory = ({
 
   const usersId = useMemo(() => uniq(data?.map(version => version.userId)), [data]);
   const hasUsersViewPerm = stripes.hasPerm('ui-users.view');
+  const centralTenantId = stripes.user.user.consortium?.centralTenantId;
 
-  const { users } = useUsersBatch(usersId);
+  const { users } = useUsersBatch(usersId, { tenantId: centralTenantId });

```

Comment:
Maybe we should check if a user has central tenant permissions? Because if they don't then they won't see any users and not just central tenant users

Reply:
@BogdanDenis all tenants will have the `users.collection.get` permission from [MARC Authority: View MARC authority record](https://github.com/folio-org/ui-marc-authorities/pull/455/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519).

Reply:
@BogdanDenis I got it. We can check if a user has central tenant permissions, what is your expectation if they don't have such permission?

Reply:
@BogdanDenis if a user doesn't have `MARC Authority: View MARC authority record` permission in the central tenant, they won't be able to open the detailed view of a shared record in a member tenant anyway, since some requests are sent to the central tenant. They also will not be able to retrieve the shared record's history in the member tenant without having `MARC Authority: View MARC authority record` in the central tenant. Checking the central tenant's `users.collection.get` permission seems excessive.